payu/models/um.py

```diff
@@ -109,19 +110,29 @@ def collate(self):
         pass
 
     def setup(self):
+        # Raise a depreciation error if the um_env.yaml file is not found
```

Comment:
I think you mean `deprecated`/`deprecation` rather than `depreciated`/`depreciation` (and it's variants). This shows up in a few variable names and comments in these changes, so I'll just use this comment as a general call-to-action.

Reply:
Good point! Just fixed this up